docs/running-on-yarn.md

```diff
@@ -793,6 +793,15 @@ The following extra configuration options are available when the shuffle service
     NodeManager.
   </td>
 </tr>
+<tr>
+  <td><code>spark.yarn.shuffle.service.logs.namespace</code></td>
+  <td><code>(not set)</code></td>
+  <td>
+    A namespace which will be appended to the class name when forming the logger name to use for
+    emitting logs from the YARN shuffle service. This would throw a warning if the namespace value
+    provided contains a space, for example, "namespace test".
```

Comment:
Similar to my comment for the warning message, can we do more to explain why we encourage these naming conventions? Maybe:
```
    A namespace which will be appended to the class name when forming the logger name to use for
    emitting logs from the YARN shuffle service, like
    `org.apache.spark.network.yarn.YarnShuffleService.logsNamespaceValue`. Since some logging frameworks
    may expect the logger name to look like a class name, it's generally recommended to provide a value which
    would be a valid Java package or class name and not include spaces.
```